src/open_inwoner/components/templatetags/form_tags.py

```diff
@@ -44,6 +44,7 @@ def render_form(parser, token):
         {% endrender_form %}
 
     Variables:
+        + action: string | The action attribute of the form.
```

Comment:
I've just noticed that we already have `form_action` variable. Are they duplicates?